packages/ai/README.md

```diff
@@ -8,10 +8,15 @@ To learn more about how to use the AI SDK, check out our [API Reference](https:/
 
 ## Installation
 
-You will need Node.js 18+ and pnpm installed on your local development machine.
```

Comment:
This README is for using the AI SDK and should not be changed. We have a separate contribution guide ( CONTRIBUTING.md in root ) that should contain this information.

Reply:
Thanks, I reverted changes to this file and made a smaller set of edits to the contributing doc.